internal/venice-common/src/main/java/com/linkedin/venice/controllerapi/ControllerApiConstants.java

```diff
@@ -203,4 +203,6 @@ public class ControllerApiConstants {
   public static final String PERSONA_STORES = "persona_stores";
   public static final String PERSONA_QUOTA = "persona_quota";
   public static final String LATEST_SUPERSET_SCHEMA_ID = "latest_superset_schema_id";
+  public static final String ENABLE_DISABLED_REPLICAS = "ENABLE_DISABLED_REPLICAS";
```

Comment:
nitpick: other constants use lowercase. Might just lowercase to keep this style consistent?

Reply:
sure